torch/distributed/fsdp/_utils.py

```diff
@@ -2,13 +2,47 @@
 
 import torch
 from torch.nn.modules.batchnorm import _BatchNorm
+from torch.nn.modules.normalization import LayerNorm
 from torch.utils._mode_utils import no_dispatch
 
+from torch.distributed.utils import _apply_to_tensors
+from functools import partial
+
 
 def _contains_batchnorm(module):
     return any(isinstance(mod, _BatchNorm) for mod in module.modules())
 
 
+def _contains_module(root, module_to_check):
+    return any(isinstance(mod, module_to_check) for mod in root.modules())
+
+def _override_module_mixed_precision(root, module_to_override, wrap_override_dict={"mixed_precision": None}):
+    for mod in root.modules():
+        if isinstance(mod, module_to_override):
+            mod._wrap_overrides = wrap_override_dict  # type: ignore[assignment]
```

Comment:
This looks unused?

Reply:
wrap_overrides is populated here and then used in auto wrapping: https://fburl.com/code/7ypnkd83